staff/carlos-diestro/app-template/profile.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Profile</title>
+</head>
+<body>
+  <header>
+    <h1>Skylab</h1>
+  </header>
+  <nav>
+    <ul>
+      <li><a href="./home.html">home</a></li>
+      <li><a href="./news.html">news</a></li>
+      <li><a href="./profile.html">profile</a></li>
+      <li><a href="./about.html">about</a></li>
+      <li><a href="./contact.html">contact</a></li>
+      <li><a href="./login.html">login</a></li>
+      <li><a href="./register.html">register</a></li>
+    </ul>
+  </nav>
+  <main>
+    <h2>Your profile</h2>
+    <section>
+      <b>Name:</b> Carlos
```

Comment:
Why not using address?

Reply:
It's only a dumb text, but I'm not sure that addres tag will be ok here

Reply:
I think @almuitur is right. 
"The address tag defines the contact information for the author/owner of a document or an article."
https://www.w3schools.com/tags/tag_address.asp

Reply:
Done